### ISSUE TITLE
[Serious] Visible label missing

### ISSUE THREAD

**Issue by ntallman** [APTrust/dart#639]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87828 |
| **Checkpoint** | 3.3 Input Assistance › Visible Labels (3.3.2.a) |
| **Impact** | Serious |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 05 Create New Workflow (Workflow) |
| **URL** | <https://aptrust.org/documentation-page/dart-digital-archivists-resource-tool/> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Date Reported** | 05/13/26 at 8:45 am |
| **Issue Status** | Open |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | kim.casey@ablr360.com |

## Description

No visual label is present and the purpose of this field is not clear without a visual label.
When you select a Workflow to view, then Export, the code to copy form field does not indicate what it is. It would be helpful to have this labeled something along the lines of Export code to copy. Related to issue #87827

## Recommendation

Rule : 
Labels MUST be visible.

How To Fix: 
Fix this issue by ensuring that the form field has a visible label and it is always visible. It is permissible for labels to appear only when a field is hovered and focused. For example, the "Select all messages" checkbox on the Gmail Inbox screen. Keep in mind, this may not be a viable solution for mobile web.

Ensure that the label is programmatically associated with the form fields via the label/for technique or ARIA attributes.

Reference: 
W3C-WAI tutorial: https://www.w3.org/WAI/tutorials/forms/

Background: 
Filling out forms correctly can be one of the more time consuming and frustrating online user experiences, and it can be even more challenging for people with disabilities. The single easiest way to prevent user errors is by providing clear labels and instructions that are available to everyone at all times. Labels and instructions should: 1) be clear and informative, 2) disclose any constraints such as required data formats or ranges, and 3) identify required fields. Well-designed labels and instructions provide enough information so people can fill out forms without undue confusion or errors.

## Screenshots

![05_visible label](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87828/05_visible_label.png)
